commander.js

```diff
@@ -6,7 +6,7 @@ const register = (server, options) => {
   if (fs.existsSync(commandDir)) {
```

Comment:
we don't need this anymore

Reply:
it's part of the plugin.  lets default command to CWD/commands and callbacks to CWD/callbacks